style/pi-hole.css

```diff
@@ -49,3 +49,9 @@ a.lookatme {
 .dataTables_scroll {
 	-webkit-overflow-scrolling: touch;
 }
+.pagination-scroll {
```

Comment:
Can we also add `-webkit-overflow-scrolling: touch;` to this element too? Aside from that, everything seems to be working as expected!